app/controllers/api/v1/orders_process_checklists_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    class OrdersProcessChecklistsController < Api::V1::ApiController
+      authorize_resource :orders_process_checklist, parent: false
+      load_resource :orders_process_checklist, only: [:show]
+
+      resource_description do
+        short "Get orders_process_checklists."
+        formats ["json"]
+        error 401, "Unauthorized"
+        error 404, "Not Found"
+        error 500, "Internal Server Error"
+      end
+
+      def index
+        @orders_process_checklists = OrdersProcessChecklist.by_order(params[:order_id])
```

Comment:
`@orders_process_checklists = @orders_process_checklists.by_order(...)`
```suggestion
        @orders_process_checklists = OrdersProcessChecklist.by_order(params[:order_id])
```

Reply:
pushing that change via github